htdocs/comm/propal/card.php

```diff
@@ -1628,7 +1628,7 @@
 	print '<td class="tdtop">' . $langs->trans('NotePublic') . '</td>';
 	print '<td valign="top">';
 	$note_public = $object->getDefaultCreateValueFor('note_public', (is_object($objectsrc)?$objectsrc->note_public:null));
-	$doleditor = new DolEditor('note_public', $note_public, '', 80, 'dolibarr_notes', 'In', 0, false, true, ROWS_3, '90%');
```

Comment:
We dont' want to introduce images when editing public notes.
The public_note can be inside a PDF or ODT. Rendering of output must match same content than output on backoffice. And having images added by the local explorer will introduce not valid links on PDF and into other features where public_notes are exposed to end user outside of a logged context